app/assets/stylesheets/content/_work_packages.sass

```diff
@@ -46,10 +46,16 @@
 
 div[class*='work-packages--details--']
   width: 100%
+  .inplace-edit--read
+    span:first-of-type
+      width: 100%
   .inplace-edit--read-value
+    width: 100%
     .dynamic-attribute
       width: 100%
 
+    div[class*='-wrapper']
```

Comment:
This one matches rather broadly, which ones do you want to select? Because now you will all all classes that call themselves some kind of wrapper.